composer.json

```diff
@@ -49,7 +49,8 @@
         "laminas/laminas-serializer": "^2.13",
         "laminas/laminas-eventmanager": "^3.5",
         "psr/container": "^1.1|^2",
-        "laminas/laminas-cli": "^1.5"
+        "laminas/laminas-cli": "^1.5",
+        "olcs/olcs-common": "^7.0.0"
```

Comment:
Just noticed this - common is only used for frontend shared code and assets - and it's an enormous repo. If there's something shared with frontend, constants or whatever, then best just copy/paste them unfortunately.